test/testresources/SmallDataCSVExample.csv

```diff
@@ -2,6 +2,6 @@ student ID,Name,Week8Exercises [6],Bookreadingandin-textexercises [5],FinalExam
 
 stu1,Will,6,5,81.5,7,0,0,10,10,10
 
-stu2,Bill,0,5,85,10,0,9,10,10,10
```

Comment:
Generally, you should not change the data in a file unless you know you are the only one using that file. Should search even comments to make sure no one else depends on that data.